scripts/run_with_beaker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+SOURCES_ARG="--source squad.latest:/squad --source glove.latest:/glove"
+RESULT_ARG="--result-path /output"
+GPU_ARG="--gpu-count=1"
+DETACH_ARG="--detach"  # or ""
+
+ECR_REPOSITORY=896129387501.dkr.ecr.us-west-2.amazonaws.com
+
+PARAM_FILE=$1
+EXPERIMENT_NAME=$2
+
+COMMIT=$(git rev-parse HEAD)
+IMAGE=$ECR_REPOSITORY/allennlp/allennlp-gpu:$COMMIT
+
+if [ ! -n "$PARAM_FILE" ] ; then
+  echo "USAGE: ./run_on_aws.sh PARAM_FILE [EXPERIMENT_NAME]"
+  exit 1
+fi
+
+if [ -n "$EXPERIMENT_NAME" ] ; then
+  EXPERIMENT_NAME_ARG="--name=$EXPERIMENT_NAME"
+else
+  EXPERIMENT_NAME_ARG=""
+fi
+
+
+set -e
+
+# Get temporary ecr login. For this command to work, you need the python awscli
+# package with a version more recent than 1.11.91.
+eval $(aws --region=us-west-2 ecr get-login --no-include-email)
+
+docker build -f Dockerfile.gpu.train -t $IMAGE . --build-arg PARAM_FILE=$PARAM_FILE
```

Comment:
You have a few options:
- Build in the config, rebuild the image each run. This is tremendously inefficient, but the most flexible.
- Mount (pull the config from a file passed in at runtime): In Beaker this would be a data set.
- Env var / command line args: Each option gets its own variable or arg.
- Config blob: Pass the json blob as an env var or command line arg. The AWS CLI does this.

I'd recommend a combination of things, depending on how much is truly configurable: Build in the most stable parts of config, and things that you change often would be environment variables or CLI args.